setup.py

```diff
@@ -39,6 +39,7 @@ def read(*paths):
     "pytest-timeout",
     "pydoc-markdown",
     "docstring-parser",
+    "yattag"
```

Comment:
We need to put all the new dependencies into the excel section of the EXTRAS_REQUIRE. And then using `helpers.import_from_plugin` will make it show a nice error is it's not installed as `fricitonless['excel']`